pwa/src/templates/templateParts/categoriesCardsAccordion/header/CategoriesCardsAccordionHeaderTemplate.module.css

```diff
@@ -58,3 +58,9 @@
 .maxNumber {
   --conduction-badge-counter-font-size: var(--web-app-badge-counter-max-number-font-size);
 }
+.categoriesCardsTitle {
+  display: flex;
+}
+.categoriesCardsBadge {
+  height: fit-content;
```

Comment:
Misschien is `fit-content` niet meer nodig met de laatste versie van de badge counter. Zou je eens de dependency willen updaten?

Door deze toevoeging:
https://github.com/nl-design-system/utrecht/blob/main/components/badge-counter/css/_mixin.scss#L18-L19